signer/src/stacks/api.rs

```diff
@@ -590,6 +650,42 @@ fn check_result<T>(client: &mut StacksClient, result: Result<T, Error>) -> Resul
 }
 
 impl StacksInteract for StacksClient {
+    async fn get_current_signer_set(
+        &mut self,
+        contract_principal: &StacksAddress,
+    ) -> Result<Vec<PublicKey>, Error> {
+        use clarity::vm::types::{BuffData, ListData, SequenceData, Value};
```

Comment:
Isn't `Value` already imported? Well, I would move these to the top of the file.

Reply:
Yeah it was. For this case I just wanted to avoid polluting the import space with low-level Clarity types, since they'll probably only be used in one or two specific cases.

Anyway, moved them up.